doc/src/manual/environment-variables.md

```diff
@@ -0,0 +1,312 @@
+[`Base.LOAD_PATH`](@ref). (In Unix-like systems, the path separator is `:`; in
+Windows systems, the path separator is `;`.) The `Base.LOAD_PATH` variable is
+where [`Base.require`](@ref) and `Base.load_in_path()` look for
+code; it defaults to the absolute paths
+
+```
+$JULIA_HOME/../local/share/julia/site/v$(Base.VERSION.major).$(Base.VERSION.minor)
+$JULIA_HOME/../share/julia/site/v$(Base.VERSION.major).$(Base.VERSION.minor)
+```
+
+so that, e. g., version 0.5 of Julia on a Linux system with a Julia executable
+at `/bin/julia` will have a default `Base.LOAD_PATH` of
+
+```
+/local/share/julia/site/v0.5
+/share/julia/site/v0.5
+```
+
+### `JULIA_PKGDIR`
+
+The path of the parent directory `Dir._pkgroot()` for the version-specific
+Julia package repositories. If the path is relative, then it is taken with
+respect to the working directory. If `$JULIA_PKGDIR` is not set, then
+`_pkgroot()` defaults to
+
+```
+$HOME/.julia
+```
+
+Then the repository location [`Base.Pkg.dir`](@ref) for a given
+Julia version is
+
+```
+$JULIA_PKGDIR/v$(Base.VERSION.major).$(Base.VERSION.minor)
+```
+
+For example, for a Linux user whose home directory is `/home/alice`, the
+directory containing the package repositories would by default be
+
+```
+/home/alice/.julia
+```
+
+and the package repository for version 0.5 of Julia would be
+
+```
+/home/alice/.julia/v0.5
+```
+
+### `JULIA_HISTORY`
+
+The absolute path `Base.find_hist_file()` of the REPL's history file. If
+`$JULIA_HISTORY` is not set, then `Base.find_hist_file()` defaults to
+
+```
+$HOME/.julia_history
+```
+
+### `JULIA_PKGRESOLVE_ACCURACY`
+
+A positive `Int` that determines how much time the max-sum subroutine
+`MaxSum.maxsum()` of the package dependency resolver
+[`Base.Pkg.resolve`](@ref) will devote to attempting
+satisfying constraints before giving up: this value is by default `1`, and
+larger values correspond to larger amounts of time.
+
+Suppose the value of `$JULIA_PKGRESOLVE_ACCURACY` is `n`. Then
+
+*   the number of pre-decimation iterations is `20*n`,
+*   the number of iterations between decimation steps is `10*n`, and
+*   at decimation steps, at most one in every `20*n` packages is decimated.
+
+## External applications
+
+### `JULIA_SHELL`
+
+The absolute path of the shell with which Julia should execute external
+commands (via `Base.repl_cmd()`). Defaults to the environment variable
+`$SHELL`, and falls back to `/bin/sh` if `$SHELL` is unset.
+
+!!! note the `fish` shell is unsupported.
+
+### `JULIA_EDITOR`, `VISUAL`, `EDITOR`
+
+The editor returned by `Base.editor()` and used in, e.g., [`Base.edit`](@ref).
+
+`$JULIA_EDITOR` takes precedence over `$VISUAL`, which in turn takes precedence
+over `$EDITOR`. If none of these environment variables is set, then the editor
+is taken to be `open` on Windows and OS X, or `/etc/alternatives/editor` if it
+exists, or `emacs` otherwise.
+
+!!! note `$JULIA_EDITOR` is *not* used in the determination of the editor for
```

Comment:
Syntax here should be

```
!!! note

    `$JULIA_EDITOR` is *not* used in the determination of the editor for
    [`Base.Pkg.edit`](@ref): this function checks `$VISUAL` and `$EDITOR`
    alone.
```

so the admonition body text needs to be written on a newline rather than running on from `!!! note`.

Same for the other admonitions in the file.